package.json

```diff
@@ -13,7 +13,7 @@
     "demo": "pnpm run --filter=demo.tutorialkit.dev dev",
     "demo:build": "pnpm run --filter=demo.tutorialkit.dev build",
     "lint": "eslint \"{packages,docs}/**/*\"",
-    "test": "pnpm run --filter=@tutorialkit/* --filter=tutorialkit test"
+    "test": "pnpm run --stream --filter=@tutorialkit/* --filter=tutorialkit test --run"
```

Comment:
Oh interesting! :eyes: What do these flags do?

Reply:
- `--run` is for Vitest to tell it to not run in watch-mode. Otherwise Vitest will stop after first package finishes run and waits for file changes. Other packages are not run at all until test run is cancelled.
- `--stream` is for pnpm to not collapse the output of the commands. 
  - <details>
    <summary>Before and after pictures</summary>
    
    No `--stream`:
    <img width="400" src="https://github.com/stackblitz/tutorialkit/assets/14806298/5236d8b4-8410-4064-8c59-853d17de22a3" />
    With `--stream`:
    <img width="400" src="https://github.com/stackblitz/tutorialkit/assets/14806298/324e44d6-4e8d-4a0a-b54e-cb1967743e37" />

   </details>